src/modules/admin/navigations/types.ts

```diff
@@ -0,0 +1,42 @@
+export type ContentPickerType =
+	| 'CONTENT_PAGE'
+	| 'COLLECTION'
+	| 'ITEM'
+	| 'BUNDLE'
+	| 'DROPDOWN'
+	| 'INTERNAL_LINK'
+	| 'EXTERNAL_LINK'
+	| 'SEARCH_QUERY'
+	| 'PROJECTS'
+	| 'ANCHOR_LINK'
+	| 'PROFILE';
+
+export enum ContentPickerTypeSchema {
```

Comment:
The schema suffix was only required to be able to export the types from the typings repo under their final name without the suffix. So we might not need it here, unless we create a typings repo for the shared types between the proxy and the client.
```suggestion
export enum ContentPickerType {
```